### ISSUE TITLE
Mac: FileNotFoundError: [Errno 2] No such file or directory: 'aplay': 'aplay'

### ISSUE THREAD

**Issue by lzufalcon** [MycroftAI/mycroft-precise#12]
There is no 'aplay' in Mac OS X system, but there is a 'play', have tried a symbol link to 'play' or simply a new copy, both of them not work for 'play' can not work with the name 'aplay'.

But this works:

```
diff --git a/precise/scripts/listen.py b/precise/scripts/listen.py
index b95a7f5..be3b3f3 100755
--- a/precise/scripts/listen.py
+++ b/precise/scripts/listen.py
@@ -12,6 +12,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import os
 from os.path import join
 from random import randint
 from subprocess import Popen
@@ -51,7 +52,7 @@ def main():
     args = create_parser(usage).parse_args()
 
     def on_activation():
-        Popen(['aplay', '-q', 'data/activate.wav'])
+        Popen(['play', '-q', os.environ['VIRTUAL_ENV'] + '/../data/activate.wav'])
         if args.save_dir:
             global chunk_num
             nm = join(args.save_dir, args.save_prefix + session_id + '.' + str(chunk_num) + '.wav')  
```

The above change also make sure precise-listen work in any directory, to fix up this issue, we'd better get the player based on system type. 'play' for Mac, 'aplay' for Linux.

**Comment by lzufalcon** [comment]
This PR fixed up this issue: https://github.com/MycroftAI/mycroft-precise/pull/13

**Comment by lyuzhilinag** [comment]
My operation system is MacOS 10.14.5 and I executed the Source Install, everything went fine except 1 error 'Building wheel for pocketsphinx (setup.py) ... error...unable to execute 'swig': No such file or directory.' I thought it was due to the lack of swig package so I installed swig manually with the command 'brew install swig'. After installing swig and executing ./setup.sh again I still got another error 'error: command 'Running setup.py install for pocketsphinx ... error...gcc' failed with exit status 1'. I thought there was still something wrong with my dependency packages, how can I fix it? thank you for your reply.
